frontend/dataTab.jsx

```diff
@@ -70,6 +70,7 @@ function Data({ navigation }) {
         data={sampleData}
         renderItem={(item) => renderData(item, navigation)}
         showsVerticalScrollIndicator={false}
+        inverted="true"
```

Comment:
Implementing this inversion of the FlatList causes several issues:
1. The Data Tab initially renders at some random place down the list, not at the top
2. Pressing the status bar to initiate scroll to top does not work unless the list is scrolled slightly

Tested on iOS simulator iPhone 12 and physical iPhone 11 Pro

Reply:
Note: Instead of inverting the list, the data that we actually get from the backend will most likely have the latest data at the top anyways. So instating of inverting the FlatList, maybe just invert the data in sampleData